SparkPost/2web2ui
# FE-621 Simplify tenant configs

Includes tenant configs in the bundle

**todo**
- [x] cleanup some config generator
- [x] remove `script` tag for config from index.html or another way to avoid 404 on production. this is needed on development but not on prod. 
- [x] update/fix tests

**how to test**
- run development mode and see if it works as usual.
- create a prod build and serve it using (serve or live-server). use a production hostname like `app.sparkpost.com` by overriding `/etc/hosts` so that hostname -> tenant map works. access the site locally using the above hostname. 
- all tests pass
